curiosrc/alertmanager/task_alert.go

```diff
@@ -0,0 +1,395 @@
+package alertmanager
```

Comment:
+ // Nobody associated with this software's development has any business relationship to pagerduty.
// This is provided as a convenient trampoline to SP's alert system of choice.

Reply:
Yes. I will add this line.